App.tsx

```diff
@@ -1,11 +1,17 @@
 import { StatusBar } from "expo-status-bar";
 import { StyleSheet, Text, View } from "react-native";
 import WelcomePage from "./components/WelcomePage";
+import ClickState from "./components/ClickState";
+import EnterStateName from "./components/EnterStateName";
+import StateQuiz from "./components/StateQuiz";
 
 export default function App() {
   return (
     <View>
       <WelcomePage />
+      <ClickState />
+      <EnterStateName/>
+      <StateQuiz />
       <StatusBar style="auto" />
```

Comment:
Didn't we discuss offline to remove this? What purpose does it serve to have four pages shown simultaneously in one screen.